connect/basic-auth-extension/src/main/java/org/apache/kafka/connect/rest/basic/auth/extension/JaasBasicAuthFilter.java

```diff
@@ -111,41 +119,12 @@ private boolean isInternalRequest(ContainerRequestContext requestContext) {
 
     public static class BasicAuthCallBackHandler implements CallbackHandler {
 
-        private static final String BASIC = "basic";
-        private static final char COLON = ':';
-        private static final char SPACE = ' ';
         private String username;
         private String password;
```

Comment:
Nit: while we're in the neighborhood, would you mind making these final?

Reply:
(Discussed [here](https://github.com/apache/kafka/pull/12846#discussion_r1024519136))